docs/design.md

```diff
@@ -4,7 +4,7 @@ Design notes
 Background
 ----------
 
-Contour uses a CRD called [`IngressRoute`][IngressRoute].  However, [ExternalDNS][]
+Contour controlls CRDs called [`IngressRoute`][IngressRoute] and [`HTTProxy`][].  However, [ExternalDNS][]
```

Comment:
See https://github.com/cybozu-go/contour-plus/blob/54886dd5848b3bc4b9ec289f74435880b7d21e20/docs/design.md

HTTPProxy is not linked correctly.